yt/chyt/controller/internal/jupyt/controller.go

```diff
@@ -17,6 +16,26 @@ const (
 )
 
 type Config struct {
+	YtAuthCookieName *string           `yson:"yt_auth_cookie_name"`
+	ExtraEnvVars     map[string]string `json:"extra_env_vars"`
+}
+
+const (
+	DefaultYtAuthCookieName = ""
+)
+
+func (c *Config) YtAuthCookieNameOrDefault() string {
+	if c.YtAuthCookieName != nil {
+		return *c.YtAuthCookieName
+	}
+	return DefaultYtAuthCookieName
+}
+
+func (c *Config) ExtraEnvVarsOrDefault() map[string]string {
+	if c.ExtraEnvVars != nil {
+		return c.ExtraEnvVars
+	}
+	return make(map[string]string)
```

Comment:
No need to allocate an empty map, just `return nil`.
In golang it's perfectly fine to iterate over a nil map (acts like an empty map).
And it's questionable whether we need `OrDefault` method for that if the default is `nil`. May be just use `c.ExtraEnvVars`?